### PR TITLE
Fixed bugs with UI

### PR DESCRIPTION
**Pull request by xMestas**
Implements #10

### CODE REVIEW

File: src/main/java/assets/game.js [hunk 1]
```diff
@@ -63,11 +63,13 @@ function registerCellListener(f) {
 }
 
 function cellClick() {
-    let row = this.parentNode.rowIndex;
-    let col = this.cellIndex;
+    let row = this.parentNode.rowIndex + 1;
```
Comment (Bensonre) [inline src/main/java/assets/game.js]: Is this because we went from 1 to 10?

File: src/main/java/cs361/battleships/models/Game.java [hunk 2]
```diff
@@ -20,11 +22,12 @@ public boolean placeShip(Ship ship, int x, char y, boolean isVertical) {
         if (!successful)
             return false;
 
+        Ship new_ship = new Ship(randKind());
```
Comment (Bensonre) [inline src/main/java/cs361/battleships/models/Game.java]: This makes sense my bad on that one, i should have caught that

File: src/main/java/cs361/battleships/models/Ship.java [hunk 3]
```diff
@@ -10,17 +10,21 @@
 	@JsonProperty private List<Square> occupiedSquares;// holds the locations on the board the is present at
 	private String kind; // holds the kind of ship that is present
 	private int length;// holds the number of spaces the ship takes up
+
+	public Ship() {
+		occupiedSquares = new ArrayList<>();
+	}
```
Comment (Bensonre) [inline src/main/java/cs361/battleships/models/Ship.java]: does he call it without arguments?

File: src/main/java/cs361/battleships/models/Game.java (hunk 2 above)
Comment (xMestas) [reply src/main/java/cs361/battleships/models/Game.java]: Nah, the starter code had this bug in it from the beginning, not you.

File: src/main/java/cs361/battleships/models/Ship.java (hunk 3 above)
Comment (xMestas) [reply src/main/java/cs361/battleships/models/Ship.java]: No, but this needs to be here for it to work for some reason.  Haven't quite figured it out why it works, but I think it has to do with when you pass the ship to a function/return a ship from the function.  Not 100 percent sure, but it works.

**Comment by xMestas** [comment]
Implements #10

**Pull request merged**